WordPress/src/main/res/values/dimens.xml

```diff
@@ -317,6 +317,8 @@
     <dimen name="quick_start_dot_size">8dp</dimen>
     <dimen name="quick_start_completed_row_height">56dp</dimen>
     <dimen name="quick_start_completed_text">14sp</dimen>
+    <dimen name="quick_start_focus_point_my_site_right_offset">24dp</dimen>
+    <dimen name="quick_start_focus_point_bottom_nav_offset">4dp</dimen>
```

Comment:
How were the `24dp` and `4dp` values determined?  The focus point seems like it's in the right place, I just wasn't sure where these numbers came from.

Reply:
I discussed this numbers with Sylvester (with a slight adjustment to `quick_start_focus_point_bottom_nav_offset` based on the actual bounding box dimensions and the way we position things).